api/ai/priority_algorithm/mutations/greedy_local_max.py

```diff
@@ -13,6 +13,14 @@
 
 
 class GreedyLocalMaxMutation(Mutation):
+    def __init__(self, num_mutations: int = 1, number_of_teams: int = 2):
+        super().__init__(num_mutations)
+        self.number_of_teams = number_of_teams
```

Comment:
nit;

unsure how it can be better, but this variable name is kinda hard to interpret - perhaps drop a comment somewhere about it, but wasn't too hard to understand from usage